sympy/solvers/ode/ode.py

```diff
@@ -332,6 +332,7 @@
     "Liouville_Integral",
     "2nd_nonlinear_autonomous_conserved",
     "2nd_nonlinear_autonomous_conserved_Integral",
+    "1st_rational_riccati",
```

Comment:
For testing while this is being worked on I would put this hint at or near the top of the list. Then it will be used more often and we can see if it causes test failures.

Also if you see the notes at the top of `test_single.py` there is a way to run the hint with all the test examples.

Reply:
I've moved it up